LEGO1/lego/legoomni/src/common/legoanimmmpresenter.cpp

```diff
@@ -466,6 +466,12 @@ MxBool LegoAnimMMPresenter::FUN_1004b6d0(MxLong p_time)
 	return TRUE;
 }
 
+// FUNCTION: LEGO1 0x1004b830
+MxBool LegoAnimMMPresenter::FUN_1004b830()
+{
+	return !(m_unk0x58 < 6);
```

Comment:
I think writing this as: `m_unk0x58 >= e_unk6` is clearer - `m_unk0x58` is some sort of state progression and this checks that it is in e_unk6 or later

Reply:
Thanks! I should have tried this. It's quite odd how the compiler chooses to compile this line.